sessions/backend-update-delete/demo-start/components/Button/index.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+const StyledButton = styled.button`
+  width: ${({ width }) => width};
+  margin: ${({ margin }) => margin};
+  appearance: none;
+  border: none;
+  background: var(--color-water-10);
+  font-size: larger;
+  padding: 0.5rem 1rem;
+  box-shadow: 0px 1px 5px -2px var(--color-granite);
+`;
+
+export default function Button({ width, margin, children, type, onClick }) {
+  return (
+    <StyledButton type={type} width={width} margin={margin} onClick={onClick}>
+      {children}
+    </StyledButton>
+  );
+}
```

Comment:
```suggestion
export default styled.button`
  width: ${({ width }) => width};
  margin: ${({ margin }) => margin};
  appearance: none;
  border: none;
  background: var(--color-water-10);
  font-size: larger;
  padding: 0.5rem 1rem;
  box-shadow: 0px 1px 5px -2px var(--color-granite);
`;
```

Reply:
Thank you for your suggestion, I've already implemented this! :) (Pretty much exactly like this, only as a named export.)

Reply:
It seems that it is not yet pushed? Because I can't find it in the commits

Reply:
Don't worry, we are still working on the demo and the session guide. The changes will follow soon - we promise. 🤞